src/common/exception/src/exception_into.rs

```diff
@@ -436,3 +437,9 @@ impl From<redis::RedisError> for ErrorCode {
         ErrorCode::DictionarySourceError(format!("Dictionary Redis Error, cause: {}", error))
     }
 }
+
+impl<T> From<PoisonError<T>> for ErrorCode {
+    fn from(error: PoisonError<T>) -> Self {
+        ErrorCode::Internal(format!("{error}"))
```

Comment:
PoisonError Perhaps we need to throw a panic and then recover the error message from the panic.